web/src/app/admin/AdminSMSSend.tsx

```diff
@@ -86,9 +92,14 @@ export default function AdminSMSSend(): JSX.Element {
                 />
               </Grid>
               <Grid item xs={12} sm={12} md={12} lg={6}>
-                <TextField
-                  onChange={(e) =>
-                    setToNumber(e.target.value.replace(/[^0-9]/g, ''))
+                <TelTextField
+                  onChange={(e: React.FormEvent<HTMLInputElement>) =>
+                    setToNumber(
+                      (e.target as HTMLInputElement).value.replace(
+                        /[^0-9]/g,
+                        '',
+                      ),
```

Comment:
Move this logic to the component itself so that someone using the component doesn't need to remember they need this for consistent behavior.